### PR TITLE
Dev/dterletskiy/virtio integration

### PR DESCRIPTION
**Pull request by dterletskiy**

### CODE REVIEW

File: meta-xt-prod-devel-rcar-driver-domain/recipes-kernel/linux/files/r8a77951-h3ulcb-domd.dts
Comment (lorc) [inline meta-xt-prod-devel-rcar-driver-domain/recipes-kernel/linux/files/r8a77951-h3ulcb-domd.dts:1]: AFAIK, H3ULCB is still supported. Why are you deleting its dts file?

File: meta-xt-prod-devel-rcar-control/recipes-guests/domd/domd.bbappend [hunk 1]
```diff
@@ -25,4 +28,9 @@ do_install:append() {
     # Call domd-set-root script
     echo "[Service]" >> ${D}${systemd_unitdir}/system/domd.service
     echo "ExecStartPre=${libdir}/xen/bin/domd-set-root" >> ${D}${systemd_unitdir}/system/domd.service
+
+    if ${@bb.utils.contains('DISTRO_FEATURES', 'enable_virtio', 'true', 'false', d)}; then
+        echo "" >> ${CFG_FILE}
+        echo "driver_domain = 1" >> ${CFG_FILE}
```
Comment (lorc) [inline meta-xt-prod-devel-rcar-control/recipes-guests/domd/domd.bbappend:34]: Is this a some new option for xl.cfg? Looks like it is temporary, as I don't believe, that any upstreamable xen tools code will have option like this

File: meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend [hunk 2]
```diff
@@ -1,12 +1,17 @@
+python () {
```
Comment (lorc) [inline meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend:1]: Stray change?

File: meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend [hunk 3]
```diff
@@ -1,12 +1,17 @@
...
+    bb.warn("@TDA: XT_DOMU_CONFIG_NAME = ", d.getVar("XT_DOMU_CONFIG_NAME", True))
+}
+
 FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 
 RDEPENDS:${PN} = "backend-ready"
-SRC_URI += "\
+SRC_URI += " \
```
Comment (lorc) [inline meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend:8]: Do you need this change?

File: meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend [hunk 4]
```diff
@@ -1,12 +1,17 @@
...
     file://domu-vdevices.cfg \
-    file://domu-pvcamera.cfg \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'pvcamera', 'file://domu-pvcamera.cfg', '', d)} \
```
Comment (lorc) [inline meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend:10]: This has nothing to do with virtio

**Review by lorc — CHANGES_REQUESTED** [review]
Could you please write comprehensive commit descriptions? 
Those are below any standards.

Also, @otyshchenko1, could you please take a look at the very first patch in this PR?

File: meta-xt-prod-devel-rcar-driver-domain/recipes-kernel/linux/files/r8a77951-h3ulcb-domd.dts
Comment (dterletskiy) [reply meta-xt-prod-devel-rcar-driver-domain/recipes-kernel/linux/files/r8a77951-h3ulcb-domd.dts:1]: I believe we did it together and you with @otyshchenko1 confirmed this deletion.

File: meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend (hunk 2 above)
Comment (dterletskiy) [reply meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend:1]: Removed

File: meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend (hunk 3 above)
Comment (dterletskiy) [reply meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend:8]: Reverted

File: meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend (hunk 4 above)
Comment (dterletskiy) [reply meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend:10]: Reverted

**Comment by dterletskiy** [comment]
> Could you please write comprehensive commit descriptions? Those are below any standards.
> 
> Also, @otyshchenko1, could you please take a look at the very first patch in this PR?

Hi @lorc.
Done

File: meta-xt-prod-devel-rcar-driver-domain/recipes-kernel/linux/files/r8a77951-h3ulcb-xen.dts
Comment (lorc) [inline meta-xt-prod-devel-rcar-driver-domain/recipes-kernel/linux/files/r8a77951-h3ulcb-xen.dts:1]: For some reason you mentioned only -domd.dts file in the commit message, but not this file. Is this a mistake?

**Pull request merged**